pages/tutorial.js

```diff
@@ -77,11 +80,12 @@ const tutorialView = (state, prev, send) => {
     },
 
     panels: [
-      blocklyHtml,
       [
-        gameHtml,
-        goalProgressHtml
-      ]
+        instructionHtml,
+        goalHtml
+      ],
+      blocklyHtml,
+      gameHtml
     ]
   })
```

Comment:
I agree with @H3rby7. I think the image is redundant. If you don't want to loose the arrows you could add them as entities into the actual game world.


What do you think about something like this. I know mad gimp skillz
![44625eec-f5d5-11e6-92a2-9b6269d0a6a0 1](https://cloud.githubusercontent.com/assets/650540/23093162/ac343170-f5db-11e6-971f-e8f825d88d6f.png)

You could actually modify the page layout, so you can set the initial size of the panels. Something like this:

``` Javascript
  const panelGroupHtml = panelGroup.component(...context)(id, {
    props: {
      panelViews: panels
    }
    initialState: {
      panelSizes: [1, 2, 2]
    }
  })
```

in [pageLayout](https://github.com/robotopia-x/robotopia/blob/ad41215a5b473ecde0aad0b851cba558c21663bb/elements/page-layout.js#L20)